packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -73,13 +73,12 @@ async function main() {
         .toString()
         .trim();
 
-    // Spin up the standalone selenium firefox
```

Comment:
I am pretty sure this file should not be changed as this is web.

Reply:
It seems to add configurable docker image

Reply:
But why this is related to this PR?

Reply:
Leo is aware; see [this](https://mendix.slack.com/archives/GV93BNVDM/p1625477560002500?thread_ts=1625477194.002200&cid=GV93BNVDM) convo